override-api-web.xml

```diff
@@ -31,7 +31,7 @@
         </init-param>
         <init-param>
             <param-name>allowedOrigins</param-name>
-            <param-value>(https://*.isaac(physics|computerscience).org|http://localhost:800[0123])</param-value>
+            <param-value>(https://*.isaacphysics.org|http://localhost:800[0123.idea]|https://isaac(physics|chemistry|biology|computerscience).eu.ngrok.io)</param-value>
```

Comment:
Revert this (James/ Meurig input) as it will break Isaac CS

Reply:
Ah, poop.